src/components/earth/Warning.tsx

```diff
@@ -0,0 +1,132 @@
+// src/components/common/WarningTooltip.tsx
+import { useState } from "react";
+import styled, { keyframes } from "styled-components";
+
+import WarningIcon from "../../assets/svgs/Warning.svg?react";
+import { rowFlex } from "../../styles/flexStyles";
+import theme from "../../styles/theme";
+
+const Warning = () => {
+  const [hovered, setHovered] = useState(false);
+
+  return (
+    <Wrapper>
+      <IconWrapper
+        onMouseEnter={() => setHovered(true)}
+        onMouseLeave={() => setHovered(false)}
+      >
+        <GlowLayer $visible={hovered} />
+        <WarningIconStyled />
+        <Minicard $visible={hovered}>
+          <Title>
+            South Korea <Emoji>🌧️</Emoji>
+          </Title>
+        </Minicard>
+      </IconWrapper>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  width: 80%;
+  height: 100%;
+  ${rowFlex({ align: "center", justify: "center" })}
+`;
+
+const IconWrapper = styled.div`
+  position: relative;
+  width: 50px;
+  height: 50px;
+  cursor: pointer;
+  svg {
+    width: 100%;
+    height: 100%;
+  }
+  &:hover > div {
+    opacity: 1;
+    transform: translateY(-12px);
+    pointer-events: auto;
+  }
+`;
+const glowFade = keyframes`
+  0% {
+    opacity: 0.6;
+    transform: scale(1);
+  }
+  50% {
+    opacity: 1;
+    transform: scale(1.2);
+  }
+  100% {
+    opacity: 0.6;
+    transform: scale(1);
+  }
+`;
+
+const GlowLayer = styled.div<{ $visible: boolean }>`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  background: radial-gradient(
+    circle,
+    rgba(255, 199, 78, 0.5) 20%,
+    transparent 70%
+  );
+  z-index: 0;
+  opacity: ${({ $visible }) => ($visible ? 1 : 0)};
+  pointer-events: none;
+  animation: ${({ $visible }) => ($visible ? glowFade : "none")} 2.5s
+    ease-in-out infinite;
+  transition: all 0.5s ease;
+`;
+
+const WarningIconStyled = styled(WarningIcon)`
+  width: 100%;
+  height: 100%;
+  z-index: 10;
+`;
+
+const Minicard = styled.div<{ $visible: boolean }>`
```

Comment:
```suggestion
const MiniCard = styled.div<{ $visible: boolean }>`
```
spellchecker에서 경고로 알려주고 있네용 🫣